CONTRIBUTING.md

```diff
@@ -1 +1,75 @@
-Please read the [contributor guidelines](https://doc.openfisca.fr/contribute/guidelines.html).
+Thank you for wanting to contribute to OpenFisca! :smiley:
+
+TL;DR: [GitHub Flow](https://guides.github.com/introduction/flow/), [SemVer](http://semver.org/), sweat on naming and messages.
+
+
+## Pull requests
+
+We follow the [GitHub Flow](https://guides.github.com/introduction/flow/): all code contributions are submitted via a pull request towards the `master` branch.
+
+Opening a Pull Request means you want that code to be merged. If you want to only discuss it, send a link to your branch along with your questions through whichever communication channel you prefer.
+
+If the Pull Request depends on another opened Pull Request on another repository (like OpenFisca-Core/OpenFisca-France), the requirements should be updated in the dependent project via its `setup.py`.
```

Comment:
No, this doesn't work.
It should be updated like [this](https://github.com/openfisca/openfisca-france/blob/master/CONTRIBUTING.md#utiliser-une-branche-spécifique-dopenfisca-core-pour-faire-passer-les-tests-dintégration-continue)